src/prototype_room_attack.js

```diff
@@ -108,14 +126,17 @@ Room.prototype.attackRoom = function() {
 
   addRoom(player, this);
 
-  if (player.level === 0) {
+  if (this.controller.level < 3 && player.level === 0) {
     attack0(this);
     player.counter++;
-    if (player.counter > 5) {
-      player.level = 1;
-      player.counter = 0;
-    }
-    Memory.players[name] = player;
+  } else if (this.controller.level < 7 && player.level < 10) {
```

Comment:
not sure I changed it now but before I had the problem that every time the attack was queued it was queued 5 times... so every time would increase counter by 5